### PR TITLE
Efolder VBMS module initial prototype test code

### PR DESCRIPTION
**Pull request by samcoforma**
Draft PR for staging testing only

### CODE REVIEW

File: modules/efolder_vbms/app/controllers/documents_controller.rb [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      ssn = @current_user.ssn
+      if !Rails.env.development?
+        c = VBMS::Client.from_env_vars(env_name: Settings.vbms.env)
+        req = VBMS::Requests::FindDocumentSeriesReference.new(ssn)
+        res = c.send_request(req)
+        #res = [[OpenStruct], [OpenStruct]]
+        dl = []
+
+        res.each do |d|
+          dl << d[0] if d[0].doc_type == '184' || d[0].doc_type == '339'
```
Comment (jerekshoe) [inline modules/efolder_vbms/app/controllers/documents_controller.rb:19]: Are we sure that we want to include `339` at this point?

File: Gemfile.lock [hunk 2]
```diff
@@ -1133,8 +1134,6 @@ DEPENDENCIES
   sentry-raven
   shrine
   sidekiq (>= 6.4.0)
-  sidekiq-ent!
```
Comment (jerekshoe) [inline Gemfile.lock:1136]: Be careful not to get rid of these 2

File: modules/efolder_vbms/lib/efolder_vbms.rb [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+      res = c.send_request(req)
+      #res = [[OpenStruct], [OpenStruct]]
+      dl = []
+      res.each do |d|
+          if d[0].doc_type == '184' or d[0].doc_type == '339'
+            dl << d[0]
+          end
+      end
+
+      dl = dl.sort_by{ |i| i.upload_date}
```
Comment (jerekshoe) [inline modules/efolder_vbms/lib/efolder_vbms.rb:20]: Make sure you do a `.reverse` on these so they are sorted from most to least recent

File: lib/claim_letters/claim_letter_downloader.rb [hunk 4]
```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+require 'claim_letters/claim_letter_test_data'
+
+module ClaimStatusTool
+  class ClaimLetterDownloader
+    def initialize(ssn)
+      @ssn = ssn
+      @client = VBMS::Client.from_env_vars(env_name: Settings.vbms.env) unless Rails.env.development? || Rails.env.test?
```
Comment (jerekshoe) [inline lib/claim_letters/claim_letter_downloader.rb:9]: I don't think you need the `unless ...` part on this

**Review by holdenhinkle — APPROVED** [review]

**Pull request merged**